core-bundle/src/Repository/MigrationRepository.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Repository;
+
+use Contao\CoreBundle\Entity\Migration;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\DBAL\Connection;
+use Symfony\Bridge\Doctrine\ManagerRegistry;
+
+/**
+ * @internal
+ *
+ * @method object|null findOneByName(string $name)
+ */
+class MigrationRepository extends ServiceEntityRepository
```

Comment:
I don't think this class is necessary at all. You can get a default repository through the doctrine registry

Reply:
I'd still use this repository and introduce an explicit `hasEntry(string $name)` method that checks for existence without the need to fetch the record (like [here](https://stackoverflow.com/questions/36801089/doctorine-check-if-record-exists-in-database#comment94987732_36802084) for example).

Reply:
Hm, couldn't really figure out how. All the documentation and examples point to this being the way to go?

Reply:
nvm, done ;)